[spec] Fix typo in shape.all_true and txN.bitmask instruction execution

HoseongLee opened the pull request:
There seems to be some typos in shape.all_true instruction and txN.bitmask execution.
The parameter is labeled as $c_1$, but the some of the ones used are just labeled $c$.

![image](https://github.com/WebAssembly/spec/assets/101983402/7f7765a3-9473-4152-b146-7cae848e6d79)

This pull request changes all the incorrect $c$ to $c_1$.

rossberg reviewed (commented):
Thanks! Instead of changing these occurrences to c_1 , can you change the other to plain c? The index is redundant in the absence of other c's.

HoseongLee commented:
I've changed all the c_1s to plain c for the above two instructions. However, the return value for txN.bitmask is also c, so I've changed it from c to i since it returns an i32.const.

rossberg reviewed (approved):
Thanks!
